test/integration/gles/replay/BUILD.bazel

```diff
@@ -62,8 +62,9 @@ go_test(
     ],
     data = [
         "//cmd/gapir/cc:gapir",
+        "//core/vulkan/vk_api_timing_layer/cc:json",
+        "//core/vulkan/vk_api_timing_layer/cc:libVkLayer_APITiming",
```

Comment:
unrelated?

Reply:
I broke the build with my last CL so this fixes it as well. (build == tests)